frontend/talentsearch/src/js/components/search/SearchContainer.tsx

```diff
@@ -339,10 +339,12 @@ const SearchContainerApi = () => {
 
   // When pools first load, they should be added to the ApplicantFilter
   useEffect(() => {
-    if (searchFormData?.pools) {
-      setApplicantFilter({ pools: searchFormData?.pools });
+    if (searchFormData?.publishedPoolAdvertisements) {
+      setApplicantFilter({
+        pools: searchFormData?.publishedPoolAdvertisements,
+      });
     }
-  }, [searchFormData?.pools]);
+  }, [searchFormData?.publishedPoolAdvertisements]);
```

Comment:
this resolves draft pools appearing on /search
it also prevents `countPoolCandidatesByPool` query from looking at draft pools